session_stats.go

```diff
@@ -0,0 +1,62 @@
+package delugeclient
+
+import (
+	"github.com/gdm85/go-rencode"
+)
+
+// SessionStats ...
+type SessionStats struct {
+	PayloadDownloadRate int64
+	PayloadUploadRate   int64
+	DownloadRate        int64
+	UploadRate          int64
+	TotalDownload       int64 // net.recv_bytes
+	TotalUpload         int64 // net.sent_bytes
+	// FreeSpace           int64
+	DhtNodes int16
+	NumPeers int16
+	// NumTorrents            int16
+	// NumPausedTorrents      int16
+	HasIncomingConnections bool
+}
+
+var sessStatsKeys = rencode.NewList(
+	"payload_download_rate",
+	"payload_upload_rate",
+	"download_rate",
+	"upload_rate",
+	"total_download",
+	"total_upload",
+	// "free_space",
+	"dht_nodes",
+	"num_peers",
+	// "num_torrents",
+	// "num_paused_torrents",
+	"has_incoming_connections",
+)
+
+// SessionStats ...
+func (c *Client) SessionStats(keys ...string) (*SessionStats, error) {
+	var args rencode.List
+	if 0 == len(keys) {
+		args.Add(sessStatsKeys)
```

Comment:
What happens when no keys are passed instead?

Reply:
It inspired from deluge.
The process in deluge.core.core like this:
```python
        if not keys:
            return self.session_status

        status = {}
        for key in keys:
                status[key] = self.session_status[key]
        return status
```

But actually these are not many and the whole return is Ok, fix it later.